.happy/requirements2.txt

```diff
@@ -0,0 +1 @@
+boto3==1.16.53
```

Comment:
what was up with the second requirements file again? Can we add a todo item on condensing these together down the road?

Reply:
Pip can't resolve conflicts between the dependencies of the modules we need to install, so we have to work around it by installing one set of packages and then the other.

Reply:
Which modules end up conflicting?

Reply:
Actually there are newer versions of our deps w/o this conflict! patched.